include/libopencm3/stm32/f3/adc.h

```diff
@@ -483,6 +483,25 @@
 #define ADC_CCR_DMACFG			(1 << 13)
 
 /* DELAY: Delay between 2 sampling phases */
+/****************************************************************************/
+/** @defgroup adc_delay ADC Delay between 2 sampling phases
+@ingroup adc_defines
+
+@{*/
+#define ADC_CCR_DELAY_1ADCCLK		(0x0 << 8)
+#define ADC_CCR_DELAY_2ADCCLK		(0x1 << 8)
+#define ADC_CCR_DELAY_3ADCCLK		(0x2 << 8)
+#define ADC_CCR_DELAY_4ADCCLK		(0x3 << 8)
+#define ADC_CCR_DELAY_5ADCCLK		(0x4 << 8)
+#define ADC_CCR_DELAY_6ADCCLK		(0x5 << 8)
+#define ADC_CCR_DELAY_7ADCCLK		(0x6 << 8)
+#define ADC_CCR_DELAY_8ADCCLK		(0x7 << 8)
+#define ADC_CCR_DELAY_9ADCCLK		(0x8 << 8)
+#define ADC_CCR_DELAY_10ADCCLK		(0x9 << 8)
+#define ADC_CCR_DELAY_11ADCCLK		(0xa << 8)
+#define ADC_CCR_DELAY_12ADCCLK		(0xb << 8)
+/**@}*/
```

Comment:
I'm not really convinced these are valuable defines, as it's just n+1.  Also, we prefer unshifted forms in general.